src/components/Text/Text.tsx

```diff
@@ -52,50 +54,32 @@ export default class Text extends React.Component<TextProps> {
   public render() {
     const { children, screenreaderText } = this.props;
 
-    if (screenreaderText === undefined) {
-      return (
-        <span className={this.getClasses()} style={this.getStyles()}>
-          {children}
-        </span>
-      );
-    }
-
     return (
-      <span className={this.getClasses()} style={this.getStyles()}>
+      <span className={this.getClasses()} style={getInlineStyles(this.props)}>
         <span aria-hidden={true}>{children}</span>
-        <ScreenReaderText>{screenreaderText}</ScreenReaderText>
+        {screenreaderText !== undefined ? <ScreenReaderText>{screenreaderText}</ScreenReaderText> : null}
       </span>
     );
   }
 
   private getClasses() {
-    const { bold, className, color, maxWidth, size, uppercase } = this.props;
-
+    const { className, size, maxWidth } = this.props;
     const classes = ['y-text'];
```

Comment:
on Text and Block components, you could use our `join` util instead of String.prototype.join. it ignores falsey values, so you don't need to check for the existence of this.props.classNames

```
const classes = ['y-text', className];
...
return join(classes);
```

i should update our yeoman component template to use this util instead of casses.join()